src/components/general/header/Header.tsx

```diff
@@ -1,16 +1,45 @@
 /* Import Dependencies */
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { useState } from 'react';
 import { Link } from 'react-router-dom';
 import { Container, Row, Col } from 'react-bootstrap';
 
+/* Import Hooks */
+import { useFetch } from 'app/Hooks';
+
 /* Import Webroot */
 import CETAFLogo from 'webroot/img/cetafLogo.png';
 
 /* Import Styles */
 import styles from './header.module.scss';
 
+/* Import Icons */
+import { faInfoCircle } from '@fortawesome/free-solid-svg-icons';
+
+/* Import API */
+import HelloWorld from 'api/HelloWorld';
+
 
-/** Component that renders the application's Header */
+/**
+ * Component that renders the application's Header
+ * @returns JSX Component
+ */
 const Header = () => {
+    /* Base variables */
+    const [apiOnline, setApiOnline] = useState<boolean | undefined>();
+
+    const fetch = useFetch();
+
+    fetch.Fetch({
```

Comment:
I understand what you are trying to do but unless you keep this on repeat (do it every X sec) I am not sure it will help much

Reply:
Changed it so when something is requested from the API like search results it sets the error variable to true or false depending on the response of the API. That should work for now as all of the pages make one or more requests to the API when loaded.